cmd/kubeadm/app/util/system/package_validator.go

```diff
@@ -127,7 +126,6 @@ func (validator *packageValidator) validate(packageSpecs []PackageSpec, manager
 		// Get the version of the package on the running machine.
 		version, err := manager.getPackageVersion(packageName)
 		if err != nil {
-			klog.V(1).Infof("Failed to get the version for the package %q: %s\n", packageName, err)
 			errs = append(errs, err)
```

Comment:
Ideally, this should be:
```
errs = append(errs, errors.Wrapf(err, "Failed to get the version of package %q", packageName))
```

Reply:
hahahahaha. So originally I did what you are saying here. BUT the error returned from `manager.getPackageVersion()` effectively says the same exact thing. I caught this because we have a test and the test failed because the error returned changed what it said. and it was mostly redundant. I can change it back to this way and update the test, it just did not seem necessary yesterday.

Reply:
Ok, let's leave it like that then.